### PR TITLE
Nutanix:virt-who config CLI cases support for SCA enabled

### PR DESCRIPTION
**Pull request by yanpliu**
Cases: PASS
```
(robottelo_vv) [root@dell-per740-68-vm-04 robottelo]# pytest ./tests/foreman/virtwho/cli/test_nutanix_sca.py --disable-pytest-warnings -q
......                                                                                                                                                                                                      [100%]
6 passed, 6 deselected, 7 warnings in 592.52s (0:09:52)
(robottelo_vv) [root@dell-per740-68-vm-04 robottelo]
```

### CODE REVIEW

File: tests/foreman/virtwho/api/test_esx.py [hunk 1]
```diff
@@ -443,7 +443,7 @@ def test_positive_deploy_configure_hypervisor_password_with_special_characters(
     def test_positive_remove_env_option(self, default_org, form_data, virtwho_config, target_sat):
         """remove option 'env=' from the virt-who configuration file and without any error
 
-        :id: 981b6828-a7ed-46d9-9c6c-9fb22af4011e
+        :id: 0674e392-6fa5-4276-8c5a-e3cc1ab6f825
```
Comment (adarshdubey-star) [inline tests/foreman/virtwho/api/test_esx.py:446]: Non blocking : Why changing the Id's here ?

Comment (yanpliu) [reply tests/foreman/virtwho/api/test_esx.py:446]: @adarshdubey-star Because when I submit the code, it showed duplicated Id's, but I did not found the duplicated IDs, but if I do not change the IDs, I can not submit the code.

File: tests/foreman/virtwho/cli/test_nutanix_sca.py [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+        :id: 565228cd-2124-41ed-89b7-84ec6ff77213
+
+        :expectedresults: hypervisor_id option can be updated.
+
+        :CaseLevel: Integration
+
+        :CaseImportance: Medium
+        """
+        values = ['uuid', 'hostname']
+        for value in values:
```
Comment (JacobCallahan) [inline tests/foreman/virtwho/cli/test_nutanix_sca.py:106]:
```suggestion
        for value in ['uuid', 'hostname']:
```

**Review by JacobCallahan — CHANGES_REQUESTED** [review]
One minor change

**Review by JacobCallahan — APPROVED** [review]

File: tests/foreman/virtwho/api/test_esx.py [hunk 3]
```diff
@@ -177,7 +177,7 @@ def test_positive_debug_option(self, default_org, form_data, virtwho_config, tar
 
         /foreman_virt_who_configure/api/v2/configs/:id"
 
-        :id: be395108-3944-4a04-bee4-6bac3fa03a19
+        :id: 24bf4c5b-cc7a-40ac-9079-e59c73984081
```
Comment (Gauravtalreja1) [inline tests/foreman/virtwho/api/test_esx.py:180]: @yanpliu Could you revert this change for the test case ids? it seems irrelevant, and resolve conflicts
